src/test/java/de/tum/in/www1/artemis/service/exam/ExamQuizServiceTest.java

```diff
@@ -1,394 +1,394 @@
-        assertThat(quizExerciseWithStatistic.getQuizPointStatistic().getParticipantsUnrated()).isZero();
-        assertThat(quizExerciseWithStatistic.getQuizPointStatistic().getParticipantsRated()).isEqualTo(numberOfParticipants);
-
-        int questionScore = quizExerciseWithStatistic.getQuizQuestions().stream().map(QuizQuestion::getPoints).reduce(0, Integer::sum);
-        assertThat(quizExerciseWithStatistic.getMaxPoints()).isEqualTo(questionScore);
-        assertThat(quizExerciseWithStatistic.getQuizPointStatistic().getPointCounters()).hasSize(questionScore + 1);
-        // check general statistics
-        for (var pointCounter : quizExerciseWithStatistic.getQuizPointStatistic().getPointCounters()) {
-            if (pointCounter.getPoints() == 0.0) {
-                assertThat(pointCounter.getRatedCounter()).isEqualTo(Math.round(numberOfParticipants / 3.0));
-                assertThat(pointCounter.getUnRatedCounter()).isZero();
-            }
-            else if (pointCounter.getPoints() == 3.0 || pointCounter.getPoints() == 4.0 || pointCounter.getPoints() == 6.0) {
-                assertThat(pointCounter.getRatedCounter()).isEqualTo(Math.round(numberOfParticipants / 6.0));
-                assertThat(pointCounter.getUnRatedCounter()).isZero();
-            }
-            else if (pointCounter.getPoints() == 7.0 || pointCounter.getPoints() == 9.0) {
-                assertThat(pointCounter.getRatedCounter()).isEqualTo(Math.round(numberOfParticipants / 12.0));
-                assertThat(pointCounter.getUnRatedCounter()).isZero();
-            }
-            else {
-                assertThat(pointCounter.getRatedCounter()).isZero();
-                assertThat(pointCounter.getUnRatedCounter()).isZero();
-            }
-        }
-        // check statistic for each question
-        for (var question : quizExerciseWithStatistic.getQuizQuestions()) {
-            if (question instanceof MultipleChoiceQuestion) {
-                assertThat(question.getQuizQuestionStatistic().getRatedCorrectCounter()).isEqualTo(Math.round(numberOfParticipants / 2.0));
-            }
-            else if (question instanceof DragAndDropQuestion) {
-                assertThat(question.getQuizQuestionStatistic().getRatedCorrectCounter()).isEqualTo(Math.round(numberOfParticipants / 3.0));
-            }
-            else {
-                assertThat(question.getQuizQuestionStatistic().getRatedCorrectCounter()).isEqualTo(Math.round(numberOfParticipants / 4.0));
-            }
-            assertThat(question.getQuizQuestionStatistic().getUnRatedCorrectCounter()).isZero();
-            assertThat(question.getQuizQuestionStatistic().getParticipantsRated()).isEqualTo(numberOfParticipants);
-            assertThat(question.getQuizQuestionStatistic().getParticipantsUnrated()).isZero();
-        }
-    }
-}
+// package de.tum.in.www1.artemis.service.exam;
```

Comment:
Why is this test disabled?

Reply:
I disabled at first as i was getting compilation errors and i pushed it commented but its enabled now.